cyberglove_trajectory/src/cyberglove_trajectory/cyberglove_mock.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2019 Shadow Robot Company Ltd - All Rights Reserved.
+# Proprietary and Confidential. Unauthorized copying of the content in this file, via any medium is strictly prohibited.
+
+import rospy
+import actionlib
+from control_msgs.msg import FollowJointTrajectoryAction, FollowJointTrajectoryGoal
+from trajectory_msgs.msg import JointTrajectoryPoint
+
+class CybergloveMock(object):
+    def __init__(self, cycle_hand_positions=True):
+        self._cycle_hand_positions = cycle_hand_positions
```

Comment:
If this is set to false, the node will do nothing at the moment. Should we just remove that param?